rust/operator-binary/src/crd/security.rs

```diff
@@ -293,42 +294,64 @@ impl KafkaTlsSecurity {
             args.push("-b".to_string());
             args.push(format!("localhost:{}", port));
             args.extend(Self::kcat_client_auth_ssl(Self::STACKABLE_TLS_KCAT_DIR));
+            args.push("-L".to_string());
         } else if self.has_kerberos_enabled() {
             let service_name = KafkaRole::Broker.kerberos_service_name();
+            let broker_port = match broker_listener_class.as_str() {
+                // for cluster-internal, kcat will connect using the broker name and the internal port
+                "cluster-internal" => port.to_string(),
+                // for other cases we will use the IP address and the externally-mapped TLS port
+                _ => node_port_cmd(STACKABLE_LISTENER_BROKER_DIR, self.client_port_name()),
+            };
+            tracing::info!("Port {broker_port}: listener {broker_listener_class}");
```

Comment:
In case of `cluster-internal`, this log statement is:

> Port 9093: listener cluster-internal

In case of `external-unstable`, it is literally:

> Port $(cat /stackable/listener-broker/default-address/ports/kafka-tls): listener external-unstable

It could be more verbose and concrete to be useful for the user.

Reply:
I've moved this log statement to level debug as it may be useful to execute that command when shelling into the container, but not enough to be an INFO statement.

Reply:
In fact, I've ditched the log statement completely as we are (now) not logging out anything that is not already in the pod yaml.